styles.css

```diff
@@ -263,6 +263,65 @@ header {
   cursor: not-allowed;
 }
 
+.contact-form-section {
+  padding: 120px 24px 124px 24px;
+}
+
+.contact-form-title-section {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+}
+
+.contact-form-title {
+  width: 327px;
+  height: 216px;
+  font-size: 32px;
+  font-weight: bold;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+}
+
+.contact-form {
+  height: 330px;
+  margin-top: 58px;
+  text-align: center;
+}
+
+.input-field {
+  height: 48px;
+  width: 100%;
+  padding: 15px 16px;
+  border: 0;
+  border-bottom: 1px solid #dbd8d7;
+}
+
+.input-textarea {
+  height: 114px;
+  background-color: #fbf8f7;
+}
+
+.footer-social-media-links {
+  display: flex;
+  justify-content: center;
+  text-align: center;
+}
+
+.footer-social-media-links li {
+  list-style: none;
+  width: 40px;
+  height: 40px;
+  margin-top: 15px;
+  margin-bottom: 13px;
+}
+
+.footer-line-section {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 8px;
+}
+
```

Comment:
Your form in the mobile version it's looking great. 💯

Reply:
Thanks, @Felipeg005.